src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -82,6 +76,9 @@ Flags:
 --access_key <string>                                     (optional)                        Set the access key for the account (default is generated)
 --secret_key <string>                                     (optional)                        Set the secret key for the account (default is generated)
 --fs_backend <none | GPFS | CEPH_FS | NFSv4>              (optional)                        Set the filesystem type of new_buckets_path (default config.NSFS_NC_STORAGE_BACKEND)
+--from_file <string>                                      (optional)                        Use details from the JSON file, there is no need to mention all the properties individually in the CLI
```

Comment:
```suggestion
--from_file <string>                                      (optional)                        Use details from the JSON file, other account properties will be ignored
```

Reply:
Changed.
We not just ignore, but we throw an error, see for example case 'should fail - cli create account using from_file with additional flags (name)'.